### PR TITLE
Switch h1s to divs

### PR DESCRIPTION
**Pull request by gkatsev**
* Switch from h1s to divs.
* add `aria-labelledby` and `aria-describedby` attributes on the tech
* update font-sizes.

### CODE REVIEW

**Comment by misteroneill** [comment]
LGTM

File: src/guid.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+let guid = 1;
+export default function newGuid () {
+  return guid++;
+};
```
Comment (dmlap) [inline src/guid.js]: It seems a *bit* silly to pull these four lines into their own file.

Comment (gkatsev) [inline src/guid.js]: but it makes the plugin file so much cleaner.

**Comment by incompl** [comment]
Come to think of it, this suggestion seems to mesh well with https://drive.google.com/a/brightcove.com/file/d/0B2rlt7RTLH11ZWpfUzA1emNlNl94TXhMRU1keEVSRWN0WXdJ/view

**Pull request merged**